src/views/index.html

```diff
@@ -27,19 +27,13 @@ <h1>COVID-19 + flights</h1>
     <div data-js-graph id="chorddiagram"></div>
 
     <div class="weekindicator">
-      <p>KW</p>
-      <p id="weekindicator-week">1</p>
+      <p id="yearindicator"></p>
```

Comment:
```suggestion
      <p data-js="yearindicator"></p>
```
Ist zwar nur stil aber ist sauberer